src/uwtools/tests/config/test_tools.py

```diff
@@ -141,6 +141,13 @@ def test_compare_configs_bad_format(caplog):
     assert msg in str(e.value)
 
 
+@pytest.mark.parametrize("fmt", ["ini", "sh"])
```

Comment:
Use data class here, too?

Reply:
Ah yes - don't even need to run both just specify it here